src/content/docs/en/guides/cms/kontent-ai.mdx

```diff
@@ -0,0 +1,472 @@
+	* Rich text field
+		* **Name:** Teaser
+		* **Element Required:** yes
+		* **Allowed in this element:** only check Text
+	* Rich text field
+		* **Name:** Content
+		* **Element Required:** yes
+	* Date & time field
+		* **Name:** Date
+	* URL slug field
+		* **Name:** URL slug
+		* **Element Required:** yes
+		* **Auto-generate from:** select "Title"
+
+Then, click on **Save Changes**.
+
+### Creating content
+
+Now, navigate to **Content & assets** tab and create a new content item of type **Blog Post**. Fill the fields using these values:
+
+* **Content item name:** Astro is amazing
+* **Title:** Astro is amazing
+* **Teaser:** Astro is an all-in-one framework for building fast websites faster.
+* **Content:** You can use JavaScript to implement the website functionality, but no client bundle is necessary.
+* **Date & time:** select today
+* **URL slug:** astro-is-amazing
+
+When you're finished, publish the blog post using the **Publish** button at the top.
+
+*Note: Feel free to create as many blog posts as you like before moving to the next step.* 
+
+### Generating content model in TypeScript
+
+Next, we'll generate TypeScript types out of your content model.
+
+*Note: This step is optional but provides a much better developer experience and allows you to discover potential problems at build time rather than at runtime.*
+
+First, install the [Kontent.ai JS model generator](https://github.com/kontent-ai/model-generator-js), [ts-node](https://github.com/TypeStrong/ts-node), and [dotenv](https://github.com/motdotla/dotenv):
+
+```shell
+npm install @kontent-ai/model-generator ts-node dotenv --save
+```
+
+Then, add the following script to package.json:
+
+```json title="package.json"
+{
+    ...
+    "scripts": {
+        ...
+        "regenerate:models": "ts-node --esm ./generate-models.ts"
+    },
+}
+```
+
+Because the types require structural information about your project that is not available in the public API, you also need to add a Content Management API key to the `.env` file. You can generate the key under **Environment settings -> API keys -> Management API**.
+
+```ini title=".env" ins={3}
+KONTENT_ENVIRONMENT_ID=YOUR_ENVIRONMENT_ID
+KONTENT_PREVIEW_API_KEY=YOUR_PREVIEW_API_KEY
+KONTENT_MANAGEMENT_API_KEY=YOUR_MANAGEMENT_API_KEY
+```
+
+And finally, add the script `generate-models.ts` that configures the model generator to generate the models:
+
+```ts title="generate-models.ts"
+import { generateModelsAsync, textHelper } from '@kontent-ai/model-generator'
+import { rmSync, mkdirSync } from 'fs'
+
+import * as dotenv from 'dotenv'
+dotenv.config()
+
+const runAsync = async () => {
+	rmSync('./src/models', { force: true, recursive: true })
+	mkdirSync('./src/models')
+
+	// change working directory to models
+	process.chdir('./src/models')
+
+	await generateModelsAsync({
+		sdkType: 'delivery',
+		apiKey: process.env.KONTENT_MANAGEMENT_API_KEY ?? '',
+		environmentId: process.env.KONTENT_ENVIRONMENT_ID ?? '',
```

Comment:
Yes, it's because it's a script ran by `ts-node`. It's the only place that uses `dotenv`, not sure if it can use another package, but I can investigate.